test-cases/longevity/longevity-mv-synchronous-updates-12h.yaml

```
@@ -0,0 +1,27 @@
+test_duration: 780
+prepare_write_cmd: ["cassandra-stress write cl=QUORUM n=10000200  -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -mode cql3 native -rate threads=1000 -col 'size=FIXED(1024) n=FIXED(5)' -pop seq=1..10000200 -log interval=15",
+                    "cassandra-stress user profile=/tmp/mv_synchronous_updates.yaml ops'(insert=25)' cl=QUORUM n=20000000 -mode cql3 native -rate threads=100",
+                    "cassandra-stress user profile=/tmp/mv_synchronous_updates_lwt.yaml ops'(insert=25)' cl=QUORUM n=1000000 -mode cql3 native -rate threads=100"
```

Is it intended to by only 1M and not 10M?
1M is very small

---

changed

---

I still see 1000000...

```suggestion
                    "cassandra-stress user profile=/tmp/mv_synchronous_updates_lwt.yaml ops'(insert=25)' cl=QUORUM n=10000000 -mode cql3 native -rate threads=100"
```

---

?

---

@roydahan 
I did not change it in purpose.
I think we do not need here a lot of rows. LWT profile is added to this test for data validation. For this goal we do not need big rows amount.